common/app/NotFound/Not-Found.jsx

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
-
 // import PropTypes from 'prop-types';
+import Link from '../Router/Link.jsx';
 
 const propTypes = {};
 
 export default function NotFound() {
   return (
-    <div>404 Not Found</div>
+    <div className='container'>
+          <div className='row'>
+            <div className='col-sm'>
+            <div className='alert alert-danger' role='alert'>
+              <h4 className='alert-heading' className='mb-12'>
+                This challenge is not found.</h4>
```

Comment:
We shouldn't tie this down to a particular route. Please use a more generic 'page not found' message.